sos/report/plugins/tftpserver.py

```diff
@@ -8,23 +8,33 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, IndependentPlugin
+from sos.report.plugins import Plugin, IndependentPlugin, PluginOpt
 
 
 class TftpServer(Plugin, IndependentPlugin):
 
     short_desc = 'TFTP Server information'
     plugin_name = 'tftpserver'
     profiles = ('sysmgmt', 'network')
-
+    services = ('tftp', )
     files = ('/etc/xinetd.d/tftp',)
-    packages = ('tftp-server',)
+    packages = ('tftp-server', 'tftpd-hpa')
+    option_list = [
+        PluginOpt('tftpboot', default=False,
+                  desc='collect content from tftpboot path')
+    ]
 
     def setup(self):
-        self.add_dir_listing([
-            '/var/lib/tftpboot',
+        self.add_copy_spec('/etc/defaultd/tftp-hpa')
```

Comment:
typo, that should be `/etc/default/tftp-hpa`

Reply:
Good catch, thanks.